Sources/MQ/Errors/Unimplemented.swift

```diff
@@ -87,6 +87,93 @@ public func unimplemented<R>(
 	}
 }
 
+/// Convenient placeholder for unimplemented function.
+///
+/// Placeholder for a function indicating unimplemented part of code.
+/// Terminates process with ``Unimplemented`` error as the cause when the function becomes executed.
+///
+/// - Parameters:
+///   - message: Message associated with this error.
+///   Default value is "Unimplemented".
+///   - file: Source code file identifier.
+///   Filled automatically based on compile time constants.
+///   - line: Line in given source code file.
+///   Filled automatically based on compile time constants.
+/// - Returns: Placeholder function which terminates the process with ``Unimplemented`` error when executed.
+public func unimplementedAsync<R>(
+	_ message: StaticString = "Unimplemented",
+	file: StaticString = #fileID,
+	line: UInt = #line
+) -> () async -> R {
+	{
+		Unimplemented
+			.error(
+				message: message,
+				file: file,
+				line: line
+			)
+			.asFatalError()
+	}
+}
+
+/// Convenient placeholder for unimplemented function.
+///
+/// Placeholder for a function indicating unimplemented throwing part of code.
+/// Throws ``Unimplemented`` when executed.
+///
+/// - Parameters:
+///   - message: Message associated with this error.
+///   Default value is "Unimplemented".
+///   - file: Source code file identifier.
+///   Filled automatically based on compile time constants.
+///   - line: Line in given source code file.
+///   Filled automatically based on compile time constants.
+/// - Returns: Function throwing ``Unimplemented`` error when executed.
+public func unimplementedThrowing<R>(
+	_ message: StaticString = "Unimplemented",
+	file: StaticString = #fileID,
+	line: UInt = #line
+) -> () throws -> R {
+	{
+		throw
+			Unimplemented
+			.error(
+				message: message,
+				file: file,
+				line: line
+			)
+	}
+}
+
+/// Convenient placeholder for unimplemented function.
+///
+/// Placeholder for a function indicating unimplemented async throwing part of code.
+/// Throws ``Unimplemented`` when executed.
+///
+/// - Parameters:
+///   - message: Message associated with this error.
+///   Default value is "Unimplemented".
+///   - file: Source code file identifier.
+///   Filled automatically based on compile time constants.
+///   - line: Line in given source code file.
+///   Filled automatically based on compile time constants.
+/// - Returns: Function throwing ``Unimplemented`` error when executed.
+public func unimplementedAsyncThrow<R>(
```

Comment:
please rename `unimplementedAsyncThrow` -> `unimplementedAsyncThrowing` - applies to all async throwing

Reply:
There were only one functions in async throwing which was labeled like that. Change was made.